tests/utils/dictionary_minifier_test.py

```diff
@@ -44,7 +46,7 @@ def testMinifyDict_whenNestedDict_shouldMinifyStringValues():
             },
             'key5': 5
         },
-        'key6': b'a ',
+        'key6': "b'", # pylint: disable=C4001
```

Comment:
Please respect the linter, you can do 'b\''

Reply:
Which does not equal the one produced by the minifiyer..